OpenXmlPowerTools/HtmlToWmlCssParser.cs

```diff
@@ -1523,11 +1521,15 @@ public bool IsColor
                         return false;
                     }
 
-                    KnownColor kc;
-                    if (Enum.TryParse(m_val, true, out kc))
+                    try
                     {
+                        Color.FromName(m_val);
```

Comment:
Is there a way to do this without a try/catch?

Reply:
I didn't find one, I agree that that's far from ideal ... Maybe we could introduce a simple caching system with a dictionary ?

Reply:
Maybe something similar to `MetricsGetter` like `ColorParser` that uses the logic from betfore (ie `Enum.Parse` but instead does `Enum.TryParse<>`)

Reply:
And to handle a not named color, do a `bool ColorParser.TryGetColor(string value, out Color color)` kind of pattern